gopkg/da-rpc/near.go

```diff
@@ -87,20 +87,6 @@ func (f *FrameRef) UnmarshalBinary(ref []byte) error {
 	return nil
 }
 
-func validateNetwork(network string) error {
```

Comment:
Great! I wonder if its worth it to add a quick comment to inform the user what it could be, just to avoid any misunderstanding that this could be an enumerated value OR a socket addr in the format [ip]:[port]

Reply:
Done 6941b55b35f66aa7d26b4c84eb6c8628ee49d32d